monkestation/code/modules/antagonists/living_lube/living_lube_abilities.dm

```diff
@@ -0,0 +1,40 @@
+/obj/effect/proc_holder/spell/aoe_turf/knock/living_lube
+	action_background_icon_state = "bg_hive" //closest to a pink spell color we have
+	invocation = "AULIE HONKSIN FIERA"
+	invocation_type = "whisper"
+
+/obj/effect/proc_holder/spell/aimed/banana_peel/living_lube
+	action_background_icon_state = "bg_hive"
+
+/obj/effect/proc_holder/spell/voice_of_god/clown/living_lube
+	action_background_icon_state = "bg_hive"
+	power_mod = 0.5 //Slightly more annoying
+
+/obj/effect/proc_holder/spell/targeted/smoke/living_lube
+	action_background_icon_state = "bg_hive"
```

Comment:
All of them use the same icon, which is going to get very confusing.
I'd suggest having someone create a set of unique sprites for the abilities.

Reply:
no the bg_hive is just the background, most of them have the same icon as the normal spells (like knock has the key)